docs/user-guide/admin-cli.md

```diff
@@ -283,6 +283,11 @@ Uuid                                  Name      Description                 Defa
 11111111-1111-1111-1111-111111111111  _DEFAULT  This is a default vmgroup
 ```
 
+In MultiNode mode, VMs from different hosts can be a part of a vmgroup.
```

Comment:
Nit: we are using "vmgroup" and "vm-group" in the same document - better to use same term consistently.